helm/boring-registry/values.yaml

```diff
@@ -116,3 +116,16 @@ server:
     #   prefix: ""
     #   saEmail: ""
     #   signedURL: false
+
+# the nginx caching proxy configuration
+proxy:
```

Comment:
Maybe go for `cachingProxy` here to avoid mixing up things with BR proxy feature.